### PR TITLE
feat: oktoWallet support

### PR DESCRIPTION
**Pull request by oktotech**
Hi Rainbow-Kit team,
This is a PR to add support for Okto wallet to Rainbow-Kit.
Okto Wallet is a simplest gateway to DeFi investment and trading needs with easy Earn & Trade features. Okto wallet allows you to trade across 1000+ tokens like SHIB, MATIC, DOGE, etc, 20+ chains like Polygon chain, Fantom chain, BSC chain, etc, and 100+ passive earning protocols.

More about okto: [https://okto.tech/](https://okto.tech/)

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.svg
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.svg:1]: Please use a square logo (only the icon) and resize to 28 x 28 and minify the SVG with svgo

File: packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          const uri = await getWalletConnectUri(
+            connector,
+            walletConnectVersion
+          );
+          return isAndroid() ? uri : `okto://wc?uri=${encodeURIComponent(uri)}`;
+        },
+      },
+      qrCode: {
+        getUri: async () =>
+          getWalletConnectUri(connector, walletConnectVersion),
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts:64]: Missing an await; please test your connector with the example deployment

**Review by DanielSinclair — CHANGES_REQUESTED** [review]

File: packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts (hunk 1 above)
Comment (oktotech) [reply packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts:64]: Hi @DanielSinclair thanks for the review. The code works in the example deployment. There exists similar implementation for `getUri` without `await` in [argentWallet](https://github.com/rainbow-me/rainbowkit/blob/2d9c1c14c88e947116ea726f5f3138c6ced95631/packages/rainbowkit/src/wallets/walletConnectors/argentWallet/argentWallet.ts#L65) and [okxWallet](https://github.com/rainbow-me/rainbowkit/blob/2d9c1c14c88e947116ea726f5f3138c6ced95631/packages/rainbowkit/src/wallets/walletConnectors/okxWallet/okxWallet.ts#L97).

Comment (DanielSinclair) [reply packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts:64]: @oktotech Please use a square SVG

Comment (oktotech) [reply packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts:64]: @DanielSinclair is it okay if we give a white square SVG with okto logo in centre. Or its Rainbow-Kit guidelines to provide a gradient filled square logo?

Comment (oktotech) [reply packages/rainbowkit/src/wallets/walletConnectors/oktoWallet/oktoWallet.ts:64]: @DanielSinclair We have updated the icon according to guidelines

**Comment by oktotech** [comment]
@DanielSinclair is there any more feedback that we can work on? Or can we get it merged?

**Comment by DanielSinclair** [comment]
When attempting to QA, I found that the wallet is not available in the US. What regions is this wallet available in?

**Comment by oktotech** [comment]
@DanielSinclair,
We have made the Ethereum chain live as well, it was the requirement for RainbowKit integration test. 
You can follow the steps as below to perform the QA:

1. Login to Okto app: Create a new wallet
2. Open the DApp in Web browser/ Mobile browser
3. Connect Wallet: Look for a button that says something like "Connect Wallet" in the DApp.
4. Scan QR Code: The DApp will show a QR code. Use your Okto app to scan it.
5. Confirm Connection: Your wallet app will show the DApp's name. Confirm the connection.
6. Use the DApp: You can now do things like trade tokens, Buy NFT
7. Approve Transactions: If you make transactions, your wallet will ask for approval.
8. Disconnect: Remember to disconnect your wallet when you're done.

Our app is available in India and other SEA countries, you can try to use VPN to access it on Android. Can you please share your email, we can connect to help your team with QA

**Comment by oktotech** [comment]
Hi @DanielSinclair,
Please let us know if there is something we can help with to expedite this PR integration.

**Comment by DanielSinclair** [comment]
@oktotech Hi, am looking to get this merged. Can you share the iOS app with me via TestFlight so that I can QA? Unfortunately you cannot use VPNs via to access apps available in other regions.

**Comment by DanielSinclair** [comment]
@oktotech I am currently seeing an issue with the WalletConnect pairing flow. Please QA against our test deployment. I will proceed to merge the connector, but note that it is not currently functional for end-users.
IMG_7884
IMG_7885

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**